README.md

```diff
@@ -1,6 +1,7 @@
 # ImmutableRecord
 
-TODO: Write a gem description
+ImmutableRecord provides simple immutable data structures with accessor
+methods and ActiveRecord style mass-assignment initialization.
```

Comment:
Well, I used "mass assignment" because that's the standard term for what AR does. And we assign it here too, it just happens only once. :-)